pyxplr/explore_outliers.py

```diff
@@ -1,29 +1,46 @@
-def explore_outliers(dataframe):
+import pandas as pd
+
+def explore_outliers(df):
     """
     Explores outliers in each feature of dataset based on standard deviation. 
-
     Arguments
     ---------
     DataFrame : pandas.DataFrame
         The target dataframe to explore
-
     Returns
     -------
-    DataFrame :
-        Dataframe containing the number of outliers for each feature
-
+    DataFrame : pandas.DataFrame
+        Dataframe containing the number of outliers for each numeric feature
     Raises
     ------
-    ValueError
-        Features specification includes an invalid / non-existent feature
-
+    TypeError
+        Raises exception if the input is not pandas.DataFrame
     Notes
     -----
-    (Additional notes to be filled during development)
-
+    Does not consider non-numeric features 
     Examples
     --------
-    outliers = explore_outliers(df)
-    outliers
+    df = pd.DataFrame({'col1': [1, 2, 1.00, 3, -1, 100], 
+                   'col2': [3, 1 ,5, -2, 3, -1]})
+    explore_outliers(df)
     """
-    raise NotImplemented
\ No newline at end of file
+    if not isinstance(df, pd.DataFrame):
+        raise Exception("input type must be pandas.DataFrame")
+        
```

Comment:
what if the input data frame  contains non-numeric columns? maybe try to add a test saying dataframe contains non-numeric columns when passing such dataframe?

Reply:
good point Lori, this will be added to issue https://github.com/UBC-MDS/pyxplr/issues/30